common/djangoapps/config_models/models.py

```diff
@@ -136,9 +145,9 @@ def current(cls, *args):
         return current
 
     @classmethod
-    def is_enabled(cls):
+    def is_enabled(cls, *args):
```

Comment:
I don't think so, since `ConfigurationModel` is the abstract base for many models that have no context for Sites, such as `DashboardConfiguration`.